uaa-server/build.gradle

```diff
@@ -50,7 +50,7 @@ cargo {
     local {
         outputFile = file("$buildDir/uaa-server.log")
         installer {
-            installUrl = 'https://www-us.apache.org/dist/tomcat/tomcat-8/v8.5.35/bin/apache-tomcat-8.5.35.zip'
+            installUrl = 'https://www-us.apache.org/dist/tomcat/tomcat-8/v8.5.38/bin/apache-tomcat-8.5.38.zip'
```

Comment:
URL not working.
Here's a working one: ```https://www-us.apache.org/dist/tomcat/tomcat-8/v8.5.40/bin/apache-tomcat-8.5.40.zip```

Reply:
I changed to the archive site instead - should be more stable.

Reply:
[https://archive.apache.org/dist/tomcat/](https://archive.apache.org/dist/tomcat/) will be better